packages/smb-onboarder/src/utils/worker-upload-helper.ts

```diff
@@ -131,3 +134,28 @@ export const prepareUpdateOfficeParams = (rowData: Cell[]): UpdateOfficeParams =
     email: rowData[11].value as string, // email column
   }
 }
+
+export const prepareUpdateNegotiatorParams = (rowData: Cell[]): NegotiatorUpdateParams => {
+  const active = Boolean(rowData[5].value?.toLowerCase() === 'true')
+  return {
+    name: rowData[0].value as string, // name column
+    jobTitle: rowData[1].value as string, // jobTitle column
+    email: rowData[2].value as string, // email column
+    mobilePhone: rowData[3].value as string, // mobilePhone column
+    active: active, // active column
+    id: rowData[6]?.value as string,
+    _eTag: rowData[7]?.value as string,
```

Comment:
Can we move the those `0,1,2,3,4` indexes to a `const`
```
const HEADER_INDEXES = {
	name: 0,
	jobTitle: 1
...
}
```
So that we don't need to put comments everywhere